csrankings.csv

```diff
@@ -6579,6 +6581,7 @@ Jeremiah Blocki,Purdue University,https://www.cs.purdue.edu/homes/jblocki,KBvKjb
 Jeremy Aarons,Monash University,http://monash.academia.edu/JeremyAarons,NOSCHOLARPAGE
 Jeremy Blackburn,University of Alabama - Birmingham,https://www.uab.edu/cas/computerscience/people/faculty-directory/jeremy-blackburn,oO1GPO8AAAAJ
 Jeremy Buhler,Washington University in St. Louis,http://www.cse.wustl.edu/~jbuhler,yQXfAWMAAAAJ
+Jeremy C. Weiss,Carnegie Mellon University,http://www.andrew.cmu.edu/user/jweiss2/,BrWvx00AAAAJ

```

Comment:
Not listed in the directory for SCS.
https://www.cs.cmu.edu/directory

Reply:
SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See Jeremy's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html)